index.js

```diff
@@ -225,6 +225,13 @@
     };
   }
 
+  //  pair :: a -> b -> Pair a b
+  function pair(a) {
+    return function(b) {
+      return [a, b];
```

Comment:
I would name the parameters `x` and `y` so they don't match the names of the type variables, but I don't feel strongly about this.